bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVKeyboardInput.java

```diff
@@ -120,6 +123,7 @@ private void sendData() {
             payload.addProperty("text", sb.toString());
             payload.addProperty("replace", 0);
         }
+        logger.debug("Message [out]: {} {}", uri, payload.toString());
```

Comment:
This is redundant:
```
2022-10-30 21:08:43.430 [DEBUG] [ernal.handler.LGWebOSTVKeyboardInput] - Message [out]: ssap://com.webos.service.ime/sendEnterKey null
2022-10-30 21:08:43.439 [TRACE] [bos.internal.handler.LGWebOSTVSocket] - Message [out]: {"type":"request","id":8,"uri":"ssap://com.webos.service.ime/sendEnterKey","payload":null}
```

Reply:
This is the equivalent of the DEBUG log you got when the mouse API is used. I just added it for consistency and to have something logged when only DEBUG level is set.

Reply:
Hmm. So maybe it should be conditional of `!logger.isTraceEnabled()` to avoid redundant logging? Although it would be kind of messy as the logging for **DEBUG** and **TRACE** would be handled in different places without any obvious correlation. Depending on state, the message could still be skipped after this:
https://github.com/openhab/openhab-addons/blob/e528564bb6f401e297649a3cfe2ad868bbdfcb8e/bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVSocket.java#L331-L355

And inner logging:
https://github.com/openhab/openhab-addons/blob/e528564bb6f401e297649a3cfe2ad868bbdfcb8e/bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVSocket.java#L374-L390

Do we really want to log payload when **TRACE** is not enabled? I believe these paths are pretty covered with appropriate logging, so maybe we should revisit mouse socket logging instead?

Reply:
I have undone the adding of logs in that class.